packages/shared/components/Sidebar.svelte

```diff
@@ -42,9 +44,12 @@
     let showStakingNotification = false
 
     const profileColor = 'blue' // TODO: each profile has a different color
+    const profileProtocol = ProfileProtocol.Iota // TODO: add the real value from profile object or app constants
+    const networkIcon = profileProtocol === ProfileProtocol.Iota ? 'iota' : 'shimmer'
```

Comment:
Also you can do this:
`const networkIcon = profileProtocol.toLowerCase()
`
instead of:
`const networkIcon = profileProtocol === ProfileProtocol.Iota ? 'iota' : 'shimmer'
` 

although I think according to coding conventions enums should be lower case anyway? 
so this:
```
export enum ProfileProtocol {
    Iota = 'IOTA',
    Shimmer = 'Shimmer',
}
```
can be:
```
export enum ProfileProtocol {
    Iota = 'iota',
    Shimmer = 'shimmer',
}
```
And then just pass the enum around to the icon